e2etests/test/edge_module_events.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+process.env.DEBUG='azure-iot-*';
+
+var async = require('async');
+var uuid = require('uuid');
+var DeviceClient = require('azure-iot-device').Client;
+var Message = require('azure-iot-device').Message;
+var Mqtt = require('azure-iot-device-mqtt').Mqtt;
+
+var sendingModuleConnectionString = process.env.IOT_MODULES_SENDING_MODULE;
```

Comment:
Yes.  For now , this code just assures me that my changes are good as I iterate.  I assume this will morph into a more official E2E once we get the infra in place.

---
In reply to: [170378578](https://github.com/Azure/azure-iot-sdk-node/pull/235#discussion_r170378578) [](ancestors = 170378578)